pkgs/development/compilers/llvm/4/lldb.nix

```diff
@@ -27,6 +27,8 @@ stdenv.mkDerivation {
       cmake/modules/LLDBStandalone.cmake
     sed -i 's,"$.LLVM_LIBRARY_DIR.",${llvm}/lib ${clang-unwrapped}/lib,' \
       cmake/modules/LLDBStandalone.cmake
+
+    patch -p1 -i ${./lldb-libedit.patch}
```

Comment:
why didn't you use [`patches`](http://nixos.org/nixpkgs/manual/#ssec-patch-phase)?